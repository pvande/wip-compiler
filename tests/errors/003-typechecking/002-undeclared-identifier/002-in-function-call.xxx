fn := (i : int) => { }

x := fn(undef)
