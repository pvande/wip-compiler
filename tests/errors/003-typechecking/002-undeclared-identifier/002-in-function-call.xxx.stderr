

Error: "Declaration for that identifier was not found"
In [1;37mtests/errors/003-typechecking/002-undeclared-identifier/002-in-function-call.xxx[0m on line [1;37m3[0m

> [0;36mx := fn(undef)[0m
  [0;31m        ^^^^^ [0m
